### ISSUE TITLE
errors using pyreadstat with --standalone

### ISSUE THREAD

**Issue by gdementen** [Nuitka/Nuitka#980]
* Please check out if the develop version of Nuitka works better for you.

Did not test that as I got 0.6.12, which is not even tagged yet on Github (but does exist on pypi???) so I guess I am running develop already.

* Nuitka version, full Python version and Platform (Windows, OSX, Linux ...)

>nuitka --version
0.6.12
Python: 3.9.1 (default, Dec 11 2020, 09:29:25) [MSC v.1916 64 bit (AMD64)]
Executable: C:\soft\Miniconda3x64\envs\sas2csv-nuitka\python.exe
OS: Windows
Arch: x86_64

* How did you install Nuitka and Python

> conda create -n sas2csv-nuitka python pytz python-dateutil six
> activate sas2csv-nuitka
> pip install pandas # installs numpy too
> pip install pyreadstat
> pip install nuitka

* Many times when you get an error from Nuitka, your setup may be special

a program with just print("hello world") works fine when compiled with --standalone

* SSCCE (this is equivalent to test.py in the .zip file, sample.sas7bdat is also in the zip file):

[nuitka_testcase.zip](https://github.com/Nuitka/Nuitka/files/5951641/nuitka_testcase.zip)

```python
import pyreadstat

if __name__ == '__main__':
    df, meta = pyreadstat.read_sas7bdat('sample.sas7bdat')
    df.to_csv('sample.csv')
```
compiles fine using either --standalone or --onefile and ultimately produces the following traceback at runtime:

```python
Traceback (most recent call last):
  File "C:\Users\gdm\AppData\Local\Test\test\001~1.0\test.py", line 3, in <module>
    import pyreadstat
  File "C:\Users\gdm\AppData\Local\Test\test\001~1.0\pyreadstat\__init__.py", line 17, in <module pyreadstat>
  File "pyreadstat\pyreadstat.pyx", line 1, in init pyreadstat.pyreadstat
  File "pyreadstat\_readstat_parser.pyx", line 28, in init pyreadstat._readstat_parser
ModuleNotFoundError: No module named 'pandas'
```

Adding an explicit "import pandas" like below (test2.py in the zip file) gets me seemingly further...
```python
import pyreadstat
import pandas

if __name__ == '__main__':
    df, meta = pyreadstat.read_sas7bdat('sample.sas7bdat')
    df.to_csv('sample.csv')
```
... but ultimately crashes at runtime too with:

```python
Traceback (most recent call last):
  File "C:\Users\gdm\devel\NUITKA~1\TEST2~1.DIS\test2.py", line 3, in <module>
  File "C:\Users\gdm\devel\NUITKA~1\TEST2~1.DIS\pyreadstat\__init__.py", line 17, in <module pyreadstat>
  File "pyreadstat\pyreadstat.pyx", line 1, in init pyreadstat.pyreadstat
ModuleNotFoundError: No module named 'pyreadstat._readstat_writer'
```

I have tried copying _readstat_writer.cp39-win_amd64.pyd from pyreadstat's directory to test2.dist\pyreadstat\ with or without renaming it to _readstat_writer.pyd but it does not seem to change anything, so I am out of ideas to solve this.

* The specific PyPI names and versions

> python -m pip freeze
Nuitka==0.6.12
numpy==1.20.1
pandas==1.2.1
pyreadstat==1.0.8

**Comment by jttaggart** [comment]
Nuitka==0.6.12
numpy==1.19.5 (I had errors with pyreadstat and numpy==1.20.1)
pandas==1.2.2
pyreadstat==1.0.9

OS==Win 10 64 bit
Python==3.7

python -m nuitka --windows-disable-console --plugin-enable=multiprocessing --mingw64 SPSS_Data_Editor_v1.0.py worked with pyreadstat

**Comment by gdementen** [comment]
I did a lot more tests today. The results are mostly the same, but I got some more hindsight from the exercice:
* I need to use --plugin-enable=numpy with --standalone (I did use it previously but did not mention it)
  * As a side note, I find it kinda strange that nuitka detects the plugin is needed but does not enable it automatically. 
* As expected, Nuitka 0.6.13 solves the first "problem" (ModuleNotFoundError: No module named 'pandas'), making test.py equivalent to test2.py
* the versions of python, numpy, pandas or pyreadstat do not seem to have any effect on the result. I tried various combinations of versions: python 3.7 and 3.9, numpy 1.19.5 and 1.20.1, pandas 1.2.1 and 1.2.2, pyreadstat 1.0.8, 1.0.9 and 1.1
* as @jttaggart hinted above, the resulting binary "works" fine if I do not use --standalone (but then the result is obviously not standalone):
  * nuitka test.py
* Strangely (to me), the following invocations do not try to compile pandas et al (and the resulting binary is thus not standalone and "runs" fine but this is not what I wanted): 
  * nuitka --follow-imports test.py
  * nuitka --follow-imports --follow-stdlib test.py
* The binary resulting from the following invocations "work" too but are not standalone (as expected):
  * nuitka --follow-import-to=pyreadstat test.py
  * nuitka --follow-import-to=pyreadstat --include-package=pyreadstat test.py

**Comment by gdementen** [comment]
as far as minimal reproducers go, what I sent was not actually minimal. A minimal.py file with just:
```import pyreadstat```
is enough to reproduce the problem using: `nuitka --standalone --enable-plugin=numpy minimal.py`

```sh
> cd minimal.dist
> minimal.exe
Traceback (most recent call last):
  File "C:\Users\gdm\devel\NUITKA~1\MINIMA~1.DIS\minimal.py", line 1, in <module>
  File "C:\Users\gdm\devel\NUITKA~1\MINIMA~1.DIS\pyreadstat\__init__.py", line 17, in <module pyreadstat>
  File "pyreadstat\pyreadstat.pyx", line 1, in init pyreadstat.pyreadstat
ModuleNotFoundError: No module named 'pyreadstat._readstat_writer'
```

**Comment by kayhayen** [comment]
If you search Nuitka 0.6.13 or higher for pyreadstat, you will find where to add implicit dependencies. Can I ask you to do this and make a PR out out if @gdementen ?

**Comment by gdementen** [comment]
@kayhayen yes, I know the place but I didn't realize it was a case of implicit dependency 😞. I will try to do that then.

**Comment by kayhayen** [comment]
I think it is, and you can basically see the one importing and use that as the implicit dependency. Since it says `.pyx` in the source code, that's something that Nuitka isn't able to look into, and they don't have the `.pyi` file it would take.

**Comment by gdementen** [comment]
fixed in 0.6.14.5
